examples/latex.tex

```diff
@@ -33,6 +33,7 @@
   superscripts,
   tableCaptions,
   taskLists,
+  lineBlocks,
```

Comment:
I like to keep the options alphabetically sorted. Adding the options to the end of the list makes it impossible to have several PRs adding new options without collisions on merge.